app/graphql/graphql_devise/mutations/sign_up.rb

```diff
@@ -0,0 +1,64 @@
+module GraphqlDevise
+  module Mutations
+    class SignUp < Base
+      argument :email,                 String, required: true
+      argument :name,                  String, required: false
+      argument :password,              String, required: true
+      argument :password_confirmation, String, required: true
+      argument :confirm_success_url,   String, required: false
+      argument :config_name,           String, required: false
+
+      def resolve(email:, **attrs)
+        redirect_url = attrs.delete(:confirm_success_url)
```

Comment:
I would add kargs that are required directly in the resolve params so you don't have to do something like this